### PR TITLE
Fixes #618 Implemented Oauth (Sign in Feature) By Google API (Updated)

### LINKED ISSUE #618: Add a way for sign in to game and rate it.
### Expected results
Add a way for sign in to game and rate it.  If not lot of points can add for one player.

### Actual results
Lot of points(ratings) can add for one player. 

### Steps to reproduce the issue
Add a way for sign in to game

### Implementation/Solution Ideas
Use gmail account instead or add a way for make a new account for game.

### Would you like to work on this
Open to anyone!

### PR DESCRIPTION
**Pull request by ShubhamChaudharyy**
(https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] included OAuth signIn feature by Google API
- [x] Adjusted Navbar Items for SignIn,Signout Buttons and Email ID of User's Credentials

### Changes done in this Pull Request

- Fixes #618 

Clip of Improvements Done:

ezgif com-video-to-gif

### CODE REVIEW

**Comment by ShubhamChaudharyy** [comment]
@YashKumarVerma you can check i have updated!

**Comment by YashKumarVerma** [comment]
Good work. Can you try to squash the commits into one?

File: index.html [hunk 1]
```diff
@@ -4,7 +4,9 @@
     <meta charset="utf-8">
     <meta name="description" content="Labyrinth | Game ">
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
+    <meta name="google-signin-client_id" content="<use your own API key and remove this rotated triangle brackets while filling api>">
```
Comment (YashKumarVerma) [inline index.html]: are you sure that this is what must go in here ?

File: js/Oauth.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+function onSignIn(googleUser)
+{ console.log('hey')
```
Comment (YashKumarVerma) [inline js/Oauth.js]: do we need a `console.log('hey')` ?

Comment (ShubhamChaudharyy) [reply js/Oauth.js]: No I forgot to delete it

File: index.html (hunk 1 above)
Comment (ShubhamChaudharyy) [reply index.html]: Yup I can't deploy my api key and fill api key after removing "<>" this

**Comment by ShubhamChaudharyy** [comment]
> Good work. Can you try to squash the commits into one?

sorry this might be a silly question but i have changed 3 files so when it compares to fork then commits corresponds to number of files changed. Please guide how to change 3 commits to one

File: index.html (hunk 1 above)
Comment (ShubhamChaudharyy) [reply index.html]: > Good work. Can you try to squash the commits into one?

okay ill make another pull request

**Pull request closed without merging**